src/components/app/app.tsx

```diff
@@ -0,0 +1,49 @@
+import './app.scss';
+
+import React from 'react';
+
+import cards from '../../modules/cards';
+import Card from '../card/card';
+import SearchBar from '../searchBar/searchBar';
+
+type AppStateType = {
+  cardIsLike: boolean[];
+};
+
+export default class App extends React.Component<{}, AppStateType> {
+  constructor(props: {} | Readonly<{}>) {
```

Comment:
if you want to use class comp you should not use constructor in it, try to research it